scrapi/base/transformer.py

```diff
@@ -39,16 +39,19 @@ def _transform(self, schema, doc):
         return transformed
 
     def _transform_iterable(self, l, doc):
-        docs = []
 
         if isinstance(l[0], tuple) and len(l) == 2:
             return self._transform_args_kwargs(l, doc)
 
-        for value in l:
+        fn, values = l[-1], l[:-1]
+        args = []
+
+        for value in values:
             if isinstance(value, basestring):
-                docs.append(self._transform_string(value, doc))
+                args.append(self._transform_string(value, doc))
             elif callable(value):
-                return value(*[res for res in docs])
+                args.append(value(doc))
```

Comment:
Why was this changed to an append?

previously:
```python
if isinstance(...):
    args.append(...)
elif callable(...):
    return ...
```
currently:
```python
if isinstance(...):
    args.append(...)
elif callable(...):
    args.append(...)
return ...
```

Won't these behaviors be very different?